ReSharper.FSharp/test/src/FSharp.Tests/FSharp.Tests.fsproj

```diff
@@ -25,6 +25,7 @@
     <Compile Include="Cache\SymbolCacheTest.fs" />
     <Compile Include="Parsing\FSharpLexerTest.fs" />
     <Compile Include="Parsing\FSharpParserTest.fs" />
+    <Compile Include="Parsing\ArgumentsOwnerTest.fs" />
```

Comment:
Please move it to `service`, the `parsing` here is about creating the trees only. The test data folder then could go to `features/service`.